contracts/UFragments.sol

```diff
@@ -155,6 +183,14 @@ contract UFragments is ERC20Detailed, Ownable {
         return _gonBalances[who].div(_gonsPerFragment);
     }
 
+    /**
+     * @param who The address to query.
+     * @return The number of successful permits by the specified address.
+     */
+    function nonces(address who) public view returns (uint256) {
```

Comment:
set to external?

Reply:
kept it consistent with `balanceOf` and `totalSupply`. Worthwhile changing all of them to `external`?

Reply:
yeah - it will give a (small) gas saving

Reply:
https://github.com/ampleforth/uFragments/issues/183